frontend/packages/schema-editor/src/components/SchemaInspector/restrictions/NumberRestrictionsReducer.test.tsx

```diff
@@ -337,11 +337,11 @@ describe('NumberRestrictionsReducer', () => {
         };
         it('Returns InvalidMaxMinValue if min > max', () => {
           const result = validateMinMax({ ...oneInclState, min: 7, max: 5 });
-          expect(result).toBe(NumberRestrictionsError.InvalidMaxMinValue);
+          expect(result).toBe(NumberRestrictionsError.IntervalMustBeLargeEnough);
         });
         it('Returns InvalidMaxMinValue if min === max', () => {
           const result = validateMinMax({ ...oneInclState, min: 5, max: 5 });
-          expect(result).toBe(NumberRestrictionsError.InvalidMaxMinValue);
+          expect(result).toBe(NumberRestrictionsError.MinMustBeLessThanOrEqualToMax);
```

Comment:
Det gir ikke mening å få feilmelding som sier at verdiene kan være like når de _er_ like.
```suggestion
          expect(result).toBe(NumberRestrictionsError.MinMustBeLessThanMax);
```

Reply:
Fikset